test/parallel/test-net-better-error-messages-port-hostname.js

```diff
@@ -3,12 +3,12 @@ var common = require('../common');
 var net = require('net');
 var assert = require('assert');
 
-var c = net.createConnection(common.PORT, '...');
+var c = net.createConnection(common.PORT, '***');
```

Comment:
@trott as I tried to explain in the commit message, the `getaddrinfo()` error code when hostname is `...` is different on different systems:

| libc | hostname | `getaddrinfo()` error code |
| --- | --- | --- |
| glibc | `...` | `EAI_NONAME` |
| musl | `...` | `EAI_AGAIN` |
| glibc | `***` | `EAI_NONAME` |
| musl | `***` |  `EAI_NONAME` |

So I changed the hostname instead of testing for both error codes:
````diff
 c.on('error', common.mustCall(function(e) {
-  assert.equal(e.code, 'ENOTFOUND');
+  assert(e.code == 'EAI_NONAME' || e.code == 'EAI_AGAIN');
   assert.equal(e.port, common.PORT);
````

Also, on musl using `...` would make the test to wait for a timeoutm while using `***` would fail instantly.